server/src/signin-button.js

```diff
@@ -18,30 +18,35 @@ exports.SignInButton = class SignInButton extends React.Component {
   render() {
     if (this.state.displaySettings) {
       return <Localized id="settingsButton">
-        <a className="settings-button" href="/settings" aria-label="Settings">
+        <button className="nav-button icon-settings" tabIndex="0" title="Settings" onClick={this.clickHandler.bind(this)}>
```

Comment:
This should remain a link; it's navigation.

Reply:
Updated to button for its advantages around built-in keyboard accessibility that lets  tab between and activate using Return/Enter

Reply:
Using links navigate the web is fundamental and I don't agree we should do something else.  Also, I can tab among links and navigate by pressing Enter.  Is that not the case on macOS Firefox?

Reply:
It works once we provide href, will update to use navigate link. Thanks